### PR TITLE
chore: Add tips for token bridge deposits

### PR DESCRIPTION
**Pull request by tkernell**
- added `tip` param for token bridge deposits. goes to `claimDeposit` caller on layer. addresses bootstrapping problem where user has no trb on layer to pay gas for `claimDeposit` call. 
- use claim deposit power threshold at time of aggregate - replaced this pr: https://github.com/tellor-io/layer/pull/303
- Separate threshold for bridge withdrawals and deposits.

### CODE REVIEW

File: evm/contracts/token-bridge/TokenBridge.sol [hunk 1]
```diff
@@ -119,12 +136,24 @@ contract TokenBridge is LayerTransition{
...
     /* Internal Functions */
     /// @notice refreshes the deposit limit every 12 hours so no one can spam layer with new tokens
-    function _refreshDepositLimit() internal returns (uint256) {
+    /// @return max amount of tokens that can be deposited
+    function _refreshDepositLimit(uint256 _amount) internal returns (uint256) {
         if (block.timestamp - depositLimitUpdateTime > DEPOSIT_LIMIT_UPDATE_INTERVAL) {
             uint256 _tokenBalance = token.balanceOf(address(this));
-            if (_tokenBalance < 100 ether) {
+            if (_tokenBalance < _amount) {
                 token.mintToOracle();
```
Comment (themandalore) [inline evm/contracts/token-bridge/TokenBridge.sol:157]: is there a scenario where _tokenBalance could still be less than the amount?  Just checking here

Comment (tkernell) [reply evm/contracts/token-bridge/TokenBridge.sol:157]: If mintToOracle hasn't been called in a while, and people have done a lot of withdrawals bringing the tokenBridge balance down a lot, tokenBalance could be lower than the desired deposit amount. 

tokenBalance==100
pending mintToOracle==1000
desired deposit==100

An alternative could be to use 
`if (_tokenBalance < _amount * 5)` for deposits

and 

`if (_tokenBalance < _amount * 20)` for withdraws

File: x/bridge/keeper/keeper.go [hunk 2]
```diff
@@ -373,6 +379,23 @@ func (k Keeper) CalculateValidatorSetCheckpoint(
 	return checkpoint, nil
 }
 
+// check whether last saved validator set is too old (more than 2 weeks)
+func (k Keeper) LastSavedValidatorSetStale(ctx context.Context) (bool, error) {
```
Comment (facundomedica) [inline x/bridge/keeper/keeper.go:383]: would it be possible that we don't update the validator set in 2 weeks?

**Review by facundomedica — APPROVED** [review]
lgtm, left a question

File: x/bridge/keeper/keeper.go (hunk 2 above)
Comment (tkernell) [reply x/bridge/keeper/keeper.go:383]: it's possible, if the validator set just doesn't change by at least 5% during that 2 week period. We need this check just to make sure the bridge doesn't get frozen. If the validator set in the bridge contract is >3 weeks old, no more bridge proofs can be run until the team multisig resets the validator set in the contract. We don't really want to have to use this multisig backstop.

**Pull request merged**